source/production/F0.Minesweeper.Components/Logic/Cell/CellVisualisationManager.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
+
+namespace F0.Minesweeper.Components.Logic.Cell
+{
+	internal class CellVisualisationManager : ICellVisualisationManager
+	{
+		private static readonly Dictionary<CellStatusType, (char? content, string cssClass)> visualisationTemplates = InitializeTranslations();
```

Comment:
I didn't use the `CellVisualisation` record since it`s `Content` property is not nullable. And I honestly do not want to change that since every user of the `CellVisualisationManager` receives a valid content which can not be null. This will also change even further when we switch to using the SVGs instead of the characters because it will totally eliminate the necessity of the `Content` property. 

Initially I wanted to create an own private record for this tuple but I couldn't come up with any good names. In case you have any suggestion I'll surey incorporate that!